scripts/o3de/o3de/android_support.py

```diff
@@ -170,6 +175,13 @@ def register_setting(key: str, description: str, default: str= None,  is_passwor
 SETTINGS_EXTRA_CMAKE_ARGS       = register_setting(key='extra.cmake.args',
                                                    description='Optional string to set additional cmake arguments during the native project generation within the android gradle build process')
 
+# CARBONATED -- begin : supported a sole install-time asset pack with size up to 1.5 Gb
+SETTINGS_INTEGRATE_AAB_ASSET_PACK = register_setting(key='integrate.aab.asset.pack',
```

Comment:
Not a fan of the verb "Integrate" here, as that usually implies integration of a library or sdk.  Instead, let's rename to "aab.enable.asset.packs"

1)  Moving "aab" to the front, to create a home for addition aab related settings
2) "Enable" indicates that we are going to enable and use the feature.

Reply:
Renamed (this and related)